dev/io.openliberty.jakarta.faces.4.1/.classpath

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-    <classpathentry kind="src" path="src"/>
```

Comment:
you missed this in the impl project

Reply:
Due to the `src/META-INF/bean.xml` file, we've decided to leave this classpathentry alone.

Reply:
no need to change there is a META-INF directory under src that still exists.